mintlify/mint
# Prepend '/' to image paths

## Summary
Users have not been seeing images in the CLI. Personally I have almost always been seeing them. One of our users said that the prepending '/' has been being removed. I suspect this is the culprit so I prepended '/' to the image paths.

I updated `yarn preconfigure` and the CLI preparser. The CLI preparser won't be referencing the new `yarn preconfigure` since we have not pushed that change yet. So this is how you will have to test.

## Test Plan
1. Run `mintlify dev` on this branch locally
2. Initially the image will be broken (this won't happen after we release the newest version)
3. Make a change to the file with the broken image and save it
4. The page should refresh and the image should show.
Note: this will not be the behavior once we actually publish the new mint version. The image will not be broken on initial load because `yarn preconfigure` will be updated!
